roles/cnf_cert/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 tnf_stage: "tests"
 tnf_check_commit_sha: false
 test_network_function_repo: "https://github.com/test-network-function/cnf-certification-test"
-test_network_function_version: v5.1.0
+test_network_function_version: v5.1.1
 test_network_function_project_name: cnf-certification-test
```

Comment:
Is this variable still used?

Reply:
Yes, if no `test_network_function_version` is provided in the pipeline, it's used this one by default: https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/cnf_cert/tasks/pre-run.yml#L152-L169, which is what happens with `tnf-test-cnf` pipeline